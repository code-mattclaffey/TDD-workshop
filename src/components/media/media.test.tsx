import React from 'react';
import { render, cleanup } from '@testing-library/react';
import {
    MediaContainer,
    MediaImageContainer,
    MediaContentContainer,
} from '.';

afterEach(() => {
    cleanup();
});

describe('<MediaContainer />, MediaImageContainer & MediaContentContainer', () => {
    it('should match inline snapshot', () => {
        const { container } = render(
            <MediaContainer additionalClassNames="c-media-item">
                <MediaImageContainer additionalClassNames="c-media-item__image-container">
                    <p>Image container</p>
                </MediaImageContainer>
                <MediaContentContainer additionalClassNames="c-media-item__content-container">
                    <p>Content container</p>
                </MediaContentContainer>
            </MediaContainer>,
        );

        expect(container).toMatchSnapshot();
    });
});
