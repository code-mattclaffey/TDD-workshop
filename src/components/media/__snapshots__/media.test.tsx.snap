// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MediaContainer />, MediaImageContainer & MediaContentContainer should match inline snapshot 1`] = `
<div>
  <div
    class="o-media c-media-item"
  >
    <div
      class="o-media__img c-media-item__image-container"
    >
      <p>
        Image container
      </p>
    </div>
    <div
      class="o-media__body c-media-item__content-container"
    >
      <p>
        Content container
      </p>
    </div>
  </div>
</div>
`;
